src/components/Search/SearchResultPanel.js

```diff
@@ -22,7 +31,7 @@ export default class SearchResultPanel extends React.Component {
   state = { showLowerButton: false };
 
   showButtons = (event) => {
-    this.setState({ showLowerButton: event.nativeEvent.target.scrollTop > 0 });
+    this.setState({ showLowerButton: event.nativeEvent.target.scrollTop > 100 });
```

Comment:
I see that this was due to UX (is ugly to show the 'go to top button' right away when there are more items in the cards)